pom.xml

```diff
@@ -81,6 +81,17 @@
             <version>${workflow.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.main</groupId>
+            <artifactId>jenkins-test-harness</artifactId>
+            <version>2.18-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>19.0</version>
```

Comment:
Why is this here? It will not work—Jenkins core bundles a version of Guava, which you cannot easily override (you would have to set a plugin-first class loader, which is rather dangerous). When you add a non-plugin `compile`-scoped dependency like this you are _bundling_ that library into your plugin.